src/lib/consts.ts

```diff
@@ -0,0 +1 @@
+export const MAX_PAGE_WIDTH: number = 1700;
```

Comment:
nit: Do we need units in here?

Reply:
I think in front end it's pretty normal to assume it's pixels